docx-asciidoc-conversion/README.md

```diff
@@ -1,12 +1,13 @@
-# docx-asciidoc-conversion
+# Conversion from docx to asciidoc format
 
 ### Introduction
 
-The objective of the project is to convert the INSPIRE Technical Guidance documents from MS Word docx format to AsciiDoc format to facilitate its access and maintenance in a GitHub repository. AsciiDoc was selected over the more common Markdown format because AsciiDoc supports many advanced document features, such as automatic section numbering, that Markdown does not support.
+The objective of the procedure is to convert the INSPIRE Technical Guidance documents from MS Word docx format to AsciiDoc format to facilitate its access and maintenance in a GitHub repository. AsciiDoc was selected over the more common Markdown format because AsciiDoc supports many advanced document features, such as automatic section numbering, that Markdown does not support.
 
 An important issue encountered is that it is not possible for a generic docx to AsciiDoc converter (such as Pandoc used here) to convert all the document content as required, which is mostly due to sections with special formatting, such as TG Requirements, TG Recommendations, Conformance Classes, XML Examples or quotes. To address this issue, a Python script was developed that applies custom styling to relevant sections of AsciiDoc version of the document (obtained using Pandoc converter) based on additional style information included in a Markdown version of the document (obtained using Pandoc as well). Additionally, the Python script fixes other minor issues with the AsciiDoc document, such as section headings, numbering, TOC, etc. Moreover, some of the styles, used to identify relevant sections, are not applied consistently in the docx, so this needs to be corrected in MS Word before the conversion.
 
-The procedure was tested using the INSPIRE Metadata Technical Guidance 2.0. The procedure might change for conversion of other TGs. In this latter case related information will be added to this document. 
+The procedure was initially developed and tested using the INSPIRE Metadata Technical Guidance 2.0 and successively tested on different types of TGs (Network Services and Data Specifications). Due to the different MS Word formats used in these documents, the development of dedicated Python scripts was to automate as much as possible the conversion of all MS Word syntax. It has been verified that, in any case, a final manual revision is always needed due to the complexity of the MS Word documents.
```

Comment:
probably 'was aimed to automate as much as possible'

Reply:
The proposed change has been included.